lib/wrapbox/configuration.rb

```diff
@@ -68,6 +69,7 @@ def self.load_config(config)
         config["enable_ecs_managed_tags"],
         config["propagate_tags"],
         config["launch_instances"]&.deep_symbolize_keys,
+        config["enable_execute_command"] || nil
```

Comment:
If `enable_execute_command` is false, the option is ignored, so I suggest the following change:

```suggestion
        config["enable_execute_command"]
```